Makefile

```diff
@@ -473,7 +473,7 @@ KUSTOMIZE_VERSION ?= v5.0.3
 CONTROLLER_TOOLS_VERSION ?= v0.16.1
 GOLANGCI_LINT_VERSION ?= v1.57.2
 KIND_VERSION ?= v0.20.0
-CHAINSAW_VERSION ?= v0.2.5
+CHAINSAW_VERSION ?= v0.2.8
```

Comment:
It seems like 0.2.9 requires go 1.23 https://github.com/kyverno/chainsaw/releases/tag/v0.2.9 so we can updated only to 0.2.8